src/libraries/System.Linq.Expressions/src/System/Dynamic/Utils/ContractUtils.cs

```diff
@@ -46,28 +46,6 @@ public static void Requires([DoesNotReturnIf(false)] bool precondition, string p
             }
         }
 
-        /// <summary>
-        /// Requires the <paramref name="value"/> to be non-<c>null</c>.
-        /// </summary>
-        /// <param name="value">
-        /// The value to check for being non-<c>null</c>.
-        /// </param>
-        /// <param name="paramName">
-        /// The parameter name to use in the <see cref="ArgumentException.ParamName"/> property when an exception is thrown.
-        /// </param>
-        /// <exception cref="ArgumentNullException">
-        /// Thrown if <paramref name="value"/> is <c>null</c>.
-        /// </exception>
-        public static void RequiresNotNull(object value, string paramName)
-        {
-            Debug.Assert(!string.IsNullOrEmpty(paramName));
-
-            if (value == null)
-            {
-                throw new ArgumentNullException(paramName);
-            }
-        }
-
         /// <summary>
```

Comment:
I came here to see if this meant that we could get rid of `ContractUtils.RequiresNotNull`. 👍